dart/lib/src/noop_hub.dart

```diff
@@ -0,0 +1,51 @@
+import 'dart:async';
+
+import 'package:sentry/src/client.dart';
+import 'package:sentry/src/hub.dart';
+import 'package:sentry/src/protocol/sentry_id.dart';
+import 'package:sentry/src/protocol/level.dart';
+import 'package:sentry/src/protocol/event.dart';
```

Comment:
IDE does the imports like this but feels weird, whats the proper way?

Reply:
it didnt complain, thats why

Reply:
Normally in Dart, relative imports should be preferred https://fuchsia.dev/fuchsia-src/development/languages/dart/style#do_import_all_files_within_a_package_using_relative_paths 
( but I know that the Flutter team doesn't follow all the Dart styleguide :)  )
In IntelliJ, the autocompletion often shows both the relative and `package:` imports.  I don't know if VSCode can be configured is this way.
We could also activate this lint https://dart-lang.github.io/linter/lints/prefer_relative_imports.html which seems to not be activated with `pedantic` to enforce the rule.

Reply:
nice I created a task for it